src/Polynomials.jl

```diff
@@ -348,14 +348,24 @@ end
 div(num::Poly, den::Poly) = divrem(num, den)[1]
 rem(num::Poly, den::Poly) = divrem(num, den)[2]
 
-function ==(p1::Poly, p2::Poly)
-    if p1.var != p2.var
-        return false
-    else
-        return p1.a == p2.a
-    end
+==(p1::Poly, p2::Poly) = (p1.var == p2.var && p1.a == p2.a)
+==(p1::Poly, n::Number) = (coeffs(p1) == [n])
+==(n::Number, p1::Poly) = (p1 == n)
+
+function isapprox{T,S}(p1::Poly{T}, p2::Poly{S};
+  rtol::Real = sqrt(eps(promote_type(T,S))), atol::Real = 0)
```

Comment:
I hadn't known `Base.rtoldefaults` before. Seems like a good function! I just implemented by reading what is available with `?isapprox`. But, then, we should remove `import Base.eps` and any method overloads related to `eps` to have consistency in the package. What do you think?

Since `Polynomials` already introduces `eps` for our use (for `roots` and `gcd`), I wanted to use `eps` and the documentation provided to us (what `reltol` is for `isapprox`).